test/functionalTest/cases/1715_batch_update/batch_update_UPDATE.test

```diff
@@ -187,15 +187,15 @@ Date: REGEX(.*)
 
 05. Try to UPDATE non-existing attribute A3, see error
 ======================================================
-HTTP/1.1 472 Unknown
-Content-Length: 128
+HTTP/1.1 404 Not Found
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "FIX: 472 Unknown returned as HTTP status code by POST /v2/op/update in some cases (now using 404 Not Found)".

NTC (just a remark)